src/main/java/com/MyApp/budgetControl/domain/expense/ExpenseDTO.java

```diff
@@ -1,34 +1,28 @@
 package com.MyApp.budgetControl.domain.expense;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
 import jakarta.validation.constraints.DecimalMin;
-import jakarta.validation.constraints.Min;
 import jakarta.validation.constraints.NotBlank;
 import jakarta.validation.constraints.NotNull;
 import jakarta.validation.constraints.Size;
 import lombok.Value;
 
-import java.util.UUID;
-
 @Value
-public class Expense {
-
-    @NotNull(message = "Id is mandatory")
-    @Min(value = 101, message = "Id must be grater than or equal to 101")
-    private final long expenseId;
```

Comment:
For POST we dont want user to send entity so as I understand that's why You removed it, but for GET we probably want, I would suggest creating 2 different DTOs for ExpenseRequest and ExpenseResponse

Reply:
As you suggested, I split it into two separate DTOs